docker_setup.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+
+import os
+import os.path
+
+from os import symlink
+from os.path import exists, join
+from shlex import split
+from sys import exit
+from subprocess import call
```

Comment:
We might want to use Python's logging library, since we print a lot of information and only some of it indicates an error.